### PR TITLE
add tags to data set metadata

### PR DESCRIPTION
**Pull request by solomontang**
- Allow users to select from a list of common metadata tags for datasets or type to create their own.
- prototype for https://jira.openlattice.com/browse/APPS-3125
image
image

### CODE REVIEW

**Review by giuliacampana — APPROVED** [review]

File: src/containers/org/OrgDataSetContainer.js [hunk 1]
```diff
@@ -74,6 +75,12 @@ const CountBadge = styled(Badge)`
   margin-right: 5px;
 `;
 
+const StyledChip = styled(Chip)`
+  color: ${NEUTRAL.N700};
+  background: ${NEUTRAL.N100};
+  font-weight: 600;
+`;
```
Comment (bergieboy) [inline src/containers/org/OrgDataSetContainer.js:82]: should we just switch the default chip to match these colors?

**Review by bergieboy — APPROVED** [review]

File: src/containers/org/OrgDataSetContainer.js (hunk 1 above)
Comment (solomontang) [reply src/containers/org/OrgDataSetContainer.js:82]: hmm dunno the designs are in consistent in that regard.

**Review by anzioka — APPROVED** [review]

File: src/containers/org/OrgDataSetContainer.js (hunk 1 above)
Comment (UnsungHero97) [reply src/containers/org/OrgDataSetContainer.js:82]: I agree, we should use the chip colors. I've told Chris to standardize around that.

File: src/containers/org/OrgDataSetContainer.js [hunk 2]
```diff
@@ -116,6 +124,9 @@ const OrgDataSetContainer = ({
   const collabCount = collaborationsByDataSetId.size;
   const collabTabText = isSuccess(getCollabWithDataSetRS) ? `Collaborations (${collabCount})` : 'Collaborations';
 
+  const tags = metadata.get(TAGS, List());
+  const tagLabel = `Protections: ${tags.join(', ')}`;
```
Comment (UnsungHero97) [inline src/containers/org/OrgDataSetContainer.js:128]: We'll eventually want to support any freeform text for tags, in addition to standardized ones, so I don't think we should have the `Protections: ` prefix.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/utils/constants/index.js [hunk 3]
```diff
@@ -52,3 +52,89 @@ export const EDIT_TITLE_DESCRIPTION_UI_SCHEMA = {
...
+    label: 'Welfare and Institutions Code',
+    options: [
+      { label: 'WIC', value: 'WIC' },
+      { label: 'WIC § 4515', value: 'WIC § 4515' },
+      { label: 'WIC § 10850(f)', value: 'WIC § 10850(f)' },
+    ]
+  },
+];
+
+export const EDIT_TITLE_DESCRIPTION_FLAGS_DATA_SCHEMA = {
```
Comment (UnsungHero97) [inline src/utils/constants/index.js:88]:
```suggestion
export const EDIT_TITLE_DESCRIPTION_TAGS_DATA_SCHEMA = {
```

File: src/utils/constants/index.js [hunk 4]
```diff
@@ -52,3 +52,89 @@ export const EDIT_TITLE_DESCRIPTION_UI_SCHEMA = {
...
+          title: 'Description',
+        },
+      },
+      required: ['title', 'description'],
+    },
+  },
+
+};
+
+export const EDIT_TITLE_DESCRIPTION_FLAGS_UI_SCHEMA = {
```
Comment (UnsungHero97) [inline src/utils/constants/index.js:121]:
```suggestion
export const EDIT_TITLE_DESCRIPTION_TAGS_UI_SCHEMA = {
```

**Review by UnsungHero97 — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Pull request merged**